bundle-size/redacted.env

```diff
@@ -0,0 +1,28 @@
+# The ID of your GitHub App
+APP_ID=21431
+
+# From the GitHub App settings page
+WEBHOOK_SECRET=[redacted]
+
+PRIVATE_KEY=[redacted]
+
+# JSON value to use as the `connection` object in Knex, to connect to a PostgreSQL database
+DATABASE_CONNECTION_JSON=[redacted]
+
+# Remove the Travis IP addresses listing for local development, update based on https://docs.travis-ci.com/user/ip-addresses/ otherwise
+# TRAVIS_IP_ADDRESSES=207.254.16.35,207.254.16.36,207.254.16.37,207.254.16.38,207.254.16.39,34.66.25.221,34.66.50.208,34.66.178.120,34.66.200.49,34.68.144.114,35.184.226.236,35.188.1.99,35.188.73.34,35.192.85.2,35.192.136.167,35.192.187.174,35.193.7.13,35.193.14.140,35.202.145.110,35.202.245.105,35.222.7.205,35.224.112.202,104.154.113.151,104.154.120.187,104.198.131.58,34.66.25.221,34.66.50.208,34.66.178.120,34.66.200.49,34.68.144.114,35.184.226.236,35.188.1.99,35.188.73.34,35.192.85.2,35.192.136.167,35.192.187.174,35.193.7.13,35.193.14.140,35.202.145.110,35.202.245.105,35.222.7.205,35.224.112.202,104.154.113.151,104.154.120.187,104.198.131.58
+
+# Generate a unique token here in any way you want, see README.md for details
+TRAVIS_PUSH_BUILD_TOKEN=[redacted]
+
+# List of GitHub teams whose members can approve bundle-size changes involving multiple files changed
+FALLBACK_APPROVER_TEAMS=ampproject/wg-runtime,ampproject/wg-performance
```

Comment:
Unrelated question; will this break if there is trailing whitespace after the commas?

Reply:
Yes, it will. I didn't see a reason to make this fault-tolerant because we rarely change the .env file anyways

Reply:
SGTM